1-fp-library/src/main/scala/fplibrary/PointFree.scala

```diff
@@ -0,0 +1,33 @@
+package io.github.sentenza.hacktoberfest.fplibrary
+
+object PointFree {
+  // val ac = compose(ab, bc)
+  def compose[A, B, C](ab: A => B, bc: B => C): A => C = a => {
```

Comment:
Is this needed at all?
`compose` is already defined in `Function1`

Reply:
No, actually it's not something that will remain, it's just an alias to see the difference between normal and Kleisli composition.